PlayFabClientSDK/pom.xml

```diff
@@ -136,9 +162,25 @@
                                 <goals>
                                     <goal>jar</goal>
                                 </goals>
+                                <configuration>
+                                    <linksource>true</linksource>
+                                    <failOnError>false</failOnError>
```

Comment:
I'm going to drop this failOnError line before I merge the changes, I would rather fix the errors than hide them all.